src/utils/components/NetworkIcons/PlugCircleCheckIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+
+type PlugCircleCheckIconProps = {
+  className?: string;
+  fill?: string;
+  height?: number | string;
+  width?: number | string;
+};
+
+const PlugCircleCheckIcon: FC<PlugCircleCheckIconProps> = ({ className, fill, height, width }) => (
```

Comment:
usually you can simply use important svg , no need to wrap it in a component

Reply:
I tried that and it didn't work. Maybe I was doing it incorrectly.